### PR TITLE
#1 - Add a calculator

### PR DESCRIPTION
**Pull request by ImmortalIdiot**
Added:
* The basic calculator methods;
* A custom expression solver;
* Documentation to the methods of the Calculator.java;
* Unit tests to check the work of Calculator methods

### CODE REVIEW

**Comment by N1kySSS** [comment]
bro, we have another style of commits

File: src/test/java/CalculatorTest.java
Comment (N1kySSS) [inline src/test/java/CalculatorTest.java:1]: add empty line in the end of the file

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -1,17 +1,65 @@
 public class Calculator {
-    public int add(int a, int b){
-        //TODO inser your realisation in method add
+
+    /** Adds two integers and returns the result.
+     *
+     * @param a the first integer
+     * @param b the second integer
+     * @return the sum of a and b
+     * */
```
Comment (N1kySSS) [inline src/main/java/Calculator.java:8]:
```suggestion
      */
```

File: src/test/java/CalculatorTest.java
Comment (ImmortalIdiot) [reply src/test/java/CalculatorTest.java:1]: Done

File: src/main/java/Calculator.java (hunk 1 above)
Comment (ImmortalIdiot) [reply src/main/java/Calculator.java]: Done

**Review by N1kySSS — APPROVED** [review]

**Pull request merged**